### PR TITLE
Add general usage to readme

### PR DESCRIPTION
**Pull request by notlee**
Links to instructions on how to include Origami components. https://github.com/Financial-Times/origami/issues/59

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-autoinit bundle size difference from 2.0.4 to 199f2efedcb817686c7d4e41111ba698645d32bf
No significant bundle size differences found.

File: README.md [hunk 1]
```diff
@@ -3,13 +3,18 @@
...
 - [JavaScript](#javascript)
 - [Migration](#migration)
 - [Contact](#contact)
 - [Licence](#licence)
 
 This component comprises a standard way of firing the `o.DOMContentLoaded` event when the equivalent browser-native event fire, and will fire the Origami events even if the native ones have already been and gone, making this suitable for bundling with components that are loaded asyncronously.
 
+## Usage
+
+`o-autoinit` is included automatically by the Origami Build Service. Manual build users may include `o-autoinit` to initialise Origami components but it is not the only option. See the component [initialisation documentation](https://origami.ft.com/docs/components/initialising/) for more information.
```
Comment (notlee) [inline README.md:16]: this is another exception but could still benefit from some general usage instructions

**Comment by origamiserviceuser** [comment]
o-autoinit bundle size difference from 2.0.4 to 1ea45fb84ead60227755721e040b5e532d15f858
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-autoinit bundle size difference from 2.0.4 to 3d6d1927e1fc74b4b3123eca4bd094052cf9b210
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v2.0.5 :tada:

**Pull request merged**